internal/token.go

```diff
@@ -1293,7 +1296,7 @@ func (z *Tokenizer) readTagAttrExpression() {
 			}
 		// Handle comments, strings within attrs
 		case '/', '"', '\'':
-			if len(z.attrTemplateLiteralStack) != 0 && c == '/' {
+			if z.attrTemplateLiteralStack[z.attrExpressionStack-1] != 0 && c == '/' {
```

Comment:
☝️ I updated logic here slightly.

`attrTemplateLiteralStack` was introduced in https://github.com/withastro/compiler/pull/280 but it looks like what we want to test here is if there is an opening backtick (so currently in a template literal) not if `attrTemplateLiteralStack` is empty